src/lib/websocket.svelte.ts

```diff
@@ -47,7 +47,7 @@ export class WebSocketManager {
     async connect(address: string) {
         if (this.connected) { throw new Error("WebSocket is already connected") }
         this.webSocket = await new Promise((resolve, reject) => {
-            const webSocket = new WebSocket(`ws://${address}`)
+            const webSocket = new WebSocket(`ws://${address}`, ["spacecoffee.blimp.v1.json"])
```

Comment:
Looks good.
Although this will have to be tested after https://github.com/Space-Coffee/blimp_ground_ws_interface/pull/7 is merged.